generate-stackbrew-library.sh

```diff
@@ -11,7 +11,7 @@ declare -A debianSuite=(
 )
 defaultAlpineVersion='3.8'
 declare -A alpineVersion=(
-	[1.9]='3.6'
+	#[1.9]='3.7'
```

Comment:
"Sneaking" in this bump we probably should've done a little while ago too (making `golang:1.9-alpine` update to the latest Alpine release now).

Reply:
![image](https://user-images.githubusercontent.com/896423/42602878-479d1346-8520-11e8-8282-661290e3a28d.png)

You weren't sneaking!